gdk-pixbuf/src/auto/versions.txt

```diff
@@ -1,2 +1,2 @@
 Generated by gir (https://github.com/gtk-rs/gir @ bad7311422ee)
-from gir-files (https://github.com/gtk-rs/gir-files @ 7ebd4478b4a5)
+from gir-files (https://github.com/gtk-rs/gir-files @ 38e847e61935+)
```

Comment:
Depends on https://github.com/gtk-rs/gir-files/pull/116 before we fix the dirty submodule

Reply:
Split into https://github.com/gtk-rs/gir-files/pull/117